pkg/printer/prometheus.go

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2022 Cisco and its affiliates
+ * All rights reserved.
+**/
+
+package printer
+
+import (
+	"fmt"
+	"io"
+
+	config "github.com/get-woke/woke/pkg/config"
+	"github.com/get-woke/woke/pkg/result"
+)
+
+// Prometheus is a output format with prometheus metrics
+type Prometheus struct {
+	writer io.Writer
+	labels map[string]string
+}
+
+// NewPrometheus returns a Prometheus Printer with color optionally disabled
+func NewPrometheus(w io.Writer, c *config.Config) *Prometheus {
+	return &Prometheus{
+		writer: w,
+		labels: c.Labels,
+	}
+}
+
+func (t *Prometheus) PrintSuccessExitMessage() bool {
+	return true
+}
+
+// Print prints the file results
+func (t *Prometheus) Print(fs *result.FileResults) error {
+
+	for _, r := range fs.Results {
+		pos := fmt.Sprintf("%d:%d-%d",
+			r.GetStartPosition().Line,
+			r.GetStartPosition().Column,
+			r.GetEndPosition().Column)
+
+		labelString := ""
+		if len(t.labels) > 0 {
+			first := true
+			for k, v := range t.labels {
+				if first {
+					labelString += ", "
+					first = false
+				}
+				labelString += k + "=" + v + ", "
+			}
+			labelString = labelString[:len(labelString)-2]
+		}
```

Comment:
This logic is duplicated across prometheus.go and pushgateway.go. Can they be consolidated? I would also recommend creating a test for this functionality specifically